<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="importModel">
        <property name="text">
         <string>Импортировать модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="objectsList">
        <item>
         <property name="text">
          <string>Камера</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Модель 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Модель 2</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="locale">
         <locale language="Russian" country="Russia"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Параметры объекта</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QScrollArea" name="scrollArea">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>237</width>
                 <height>223</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <item row="1" column="1">
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLineEdit" name="objectColor">
                      <property name="text">
                       <string>ff00ff</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QPushButton" name="objectColorChoose">
                      <property name="text">
                       <string>Выбрать</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Цвет</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>Название</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="objectName">
                    <property name="text">
                     <string>Модель 1</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QTabWidget" name="tabWidget_2">
                  <property name="currentIndex">
                   <number>-1</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="deleteObject">
                  <property name="text">
                   <string>Удалить данный объект</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Параметры размытия</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBox_4"/>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>аккумулирующие</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>пиксельное</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>соседнее пиксельное</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>нет</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_44">
              <property name="text">
               <string>Тип размытия</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_45">
              <property name="text">
               <string>Количество
итераций</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_46">
              <property name="text">
               <string>Время
выдержки</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QTimeEdit" name="timeEdit_14"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Параметры вывода</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>JPG</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_39">
              <property name="text">
               <string>Разрешение
изображения</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>Qt Painter</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_42">
              <property name="text">
               <string>Движок</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QSpinBox" name="spinBox_2">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox">
                <property name="minimum">
                 <number>100</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_41">
              <property name="text">
               <string>Частота
кадров
(для видео)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBox_3">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="value">
               <number>30</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_40">
              <property name="text">
               <string>Выходной
формат</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_43">
              <property name="text">
               <string>Файл</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="text">
               <string>./image.jpg</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>Запустить вывод в файл</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rightLayout" stretch="1,0">
      <item>
       <widget class="QOpenGLWidget" name="canvas"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="playerBox">
        <item>
         <widget class="QPushButton" name="playButton">
          <property name="text">
           <string>▶</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="curentTimeSlider">
          <property name="maximum">
           <number>120000</number>
          </property>
          <property name="pageStep">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTimeEdit" name="curentTime">
          <property name="inputMethodHints">
           <set>Qt::ImhPreferNumbers</set>
          </property>
          <property name="displayFormat">
           <string>HH:mm:ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
